facility-manager.html

```diff
@@ -0,0 +1,806 @@
+---
+layout: default
+title: Using OMERO for Facility Managers
+menu-id: l1-workflow2
+pdfs:
+    - facility-manager.pdf
+description: Descriptions of common workflows for Facility Managers and links to useful sections of User Help and Developer documentation.
+version:
+    - yes
+---
+<div class="page-navigation">
+<h2>Contents</h2>
+<p>
+<a href="#introduction">Introduction</a><br />
+<a href="#using">Using OMERO</a><br />
+<a href="#permissions">Permissions on OMERO</a><br />
+<a href="#access">Access control, authentication, user accounts</a><br />
+<a href="#manage">Managing Groups</a><br />
+<a href="#import">Importing</a><br />
+<a href="#data">Data structure</a><br />
+<a href="#measure">Measurement and analysis</a><br />
+<a href="#disk">Disk usage</a><br />
+<a href="#email">Emailing users</a><br />
+<a href="#update">Updating</a><br />
```

Comment:
I would use 'upgrading' rather than updating to be consistent with our technical docs.